packages/flutter/lib/src/material/flexible_space_bar.dart

```diff
@@ -123,6 +125,11 @@ class FlexibleSpaceBar extends StatefulWidget {
   /// not centered, `EdgeInsetsDirectional.only(start: 0, bottom: 16)` otherwise.
   final EdgeInsetsGeometry? titlePadding;
 
+  /// Defines how much the title is scaled when expanded.
+  ///
+  /// Defaults to 1.5 and must be greater than 1.
+  final double expandedScale;
```

Comment:
Since this parameter only refers to the title, expandedTitleScale would make the name a little more self-explanatory.